pkg/operators/kubeipresolver/kubeipresolver.go

```diff
@@ -70,7 +73,20 @@ func (k *KubeIPResolver) Close() error {
 }
 
 func (k *KubeIPResolver) Instantiate(gadgetCtx operators.GadgetContext, gadgetInstance any, params *params.Params) (operators.OperatorInstance, error) {
-	k8sInventory, err := common.GetK8sInventoryCache()
+	config, err := k8sutil.NewKubeConfig("")
```

Comment:
The same modifications should be done to `KubeNameResolver` too:
https://github.com/inspektor-gadget/inspektor-gadget/blob/4f88e74d9225bf483d48f0cee3e658ad178c56a0/pkg/operators/kubenameresolver/kubenameresolver.go#L80

Reply:
Good catch. thanks!